velox/docs/functions/spark/map.rst

```diff
@@ -6,18 +6,18 @@ Map Functions
 
     Returns value for given ``key``, or ``NULL`` if the key is not contained in the map.
 
-.. spark:function:: map() -> map(unknown, unknown)
+.. spark:function:: map(K, V, K, V, ...) -> map(K,V)
 
-    Returns an empty map. ::
+    Returns a map created using the given key/value pairs. ::
 
-        SELECT map(); -- {}
+        SELECT map(1, 2, 3, 4); -- {1 -> 2, 3 -> 4}
```

Comment:
Can we add some examples for complex type? Maybe also clarify the expected behavior when key or value is null.

Reply:
Yes, added.